### ISSUE TITLE
Usage with Pytorch Dataset does not work for multiple input

### ISSUE THREAD

**Issue by majasomething** [stardist/augmend#6]
Trying this example:
```
from torch.utils.data import TensorDataset

data = TensorDataset(torch.tensor(x),torch.tensor(y))

data = aug.torch_wrap(data)

x2, y2 = data[0]
```

results in the following error:
"tree structures not compatible" issued from augmend\torch_utils.py line 33 (self.aug(x))

**Comment by maweigert** [comment]
Can you provide a minimal example showing the error above?

The following works for me as expected: 


```python 
import numpy as np
import augmend
import torch
from torch.utils.data import TensorDataset

x = np.zeros((10, 1, 100,100), np.float32)
y = np.zeros((10, 1, 100,100), np.float32)

data = TensorDataset(torch.tensor(x),torch.tensor(y))

aug = augmend.Augmend()
aug.add([augmend.FlipRot90(axis=(-1,-2)), augmend.AdditiveNoise()])

data = aug.torch_wrap(data)

x2, y2 = data[0]
```

**Comment by majasomething** [comment]
Thanks for your quick response. 
I think the issue was caused by the axis choice. 
When I first used it for classification with axis=(1,2) it worked. For the segmentation task, it worked with the axis set to (-1,-2) as you suggested here.
